### ISSUE TITLE
Buffer not supported on mobile devices - ExcalidrawFonts

### ISSUE THREAD

**Issue by zsviczian** [excalidraw/excalidraw#8472]
@Mrazator

Buffer class is a part of Node.js, not standard JavaScript that runs in browsers (or WebView). Should Buffer be used here?
This causes error with Excalidraw on Obsidian Mobile:

https://github.com/excalidraw/excalidraw/blob/51ea184938d57637faa345de85028aa8a0930ed2/packages/excalidraw/fonts/ExcalidrawFont.ts#L52-L55

In the other case there is a check around it.

https://github.com/excalidraw/excalidraw/blob/51ea184938d57637faa345de85028aa8a0930ed2/packages/excalidraw/fonts/ExcalidrawFont.ts#L137-L142

**Comment by zsviczian** [comment]
ah... I see why the issue happens in Obsidian. One key feature of the Obsidian Excalidraw plugin is how it is 100% standalone / local. This means I have a large main.js file that includes everything, all the fonts, packages, dependencies... everything. main.js includes the fonts as base64 data URLs. In fact, Obsidian desktop runs on Electron and provides a full Node environment, but Obsidian mobile runs in the Android/iOS webview.

So this issue is limited to Obsidian, still, would be nice to prepare the code to be dataurl font friendly.

**Comment by Mrazator** [comment]
Yeah, the assumption that dataurl === node env is wrong in this case. We don't really need to to use Buffer here - it is just a shortcut for getting ArrayBuffer from a base64.
